pom.xml

```diff
@@ -143,7 +143,7 @@
         <groupId>org.jenkins-ci.main</groupId>
         <artifactId>jenkins-war</artifactId>
         <version>${jenkins.version}</version>
-        <type>executable-war</type>
+        <type>war</type>
```

Comment:
I have no idea why this was of type `executable-war` before, but the only type actually published [here](https://repo.jenkins-ci.org/artifactory/public/org/jenkins-ci/main/jenkins-war/2.326/) is `war`.